fixtures/rabota_fixtures.py

```diff
@@ -0,0 +1,98 @@
+import re
+
+import pytest
+
+from clients.http_client import HTTPClient
+from models.parser_jobs_tut_by import RabotaByParser
+
+url = 'https://rabota.by/search/' \
+      'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=python&page=0'
+HEADERS = {
+    'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/webkit-version (KHTML, like Gecko) '
+                  'Silk / browser - version like Chrome / chrome - version Safari / webkit - version'}
+
```

Comment:
url and Headers should not here, put this vars into fixture which uses these vars

Reply:
Done